packages/utils/src/envelope.ts

```diff
@@ -134,6 +134,8 @@ const ITEM_TYPE_TO_DATA_CATEGORY_MAP: Record<EnvelopeItemType, DataCategory> = {
   client_report: 'internal',
   user_report: 'default',
   profile: 'profile',
+  replay_event: 'replay_event',
+  replay_recording: 'replay_recording',
```

Comment:
damn we should have named these `replay` and `recording` to be consistent with our other event types